src/isdimm_vpd_parser.cpp

```diff
@@ -0,0 +1,312 @@
+#include "isdimm_vpd_parser.hpp"
+
+#include <algorithm>
+#include <iostream>
+#include <numeric>
+#include <optional>
+#include <string>
+#include <unordered_map>
+
+namespace openpower

```

Comment:
We are looking for lesser level of nested namespace. Please check the existing code for reference.